MinecraftClient/Protocol/Handlers/Protocol18Terrain.cs

```diff
@@ -38,19 +39,27 @@ public Protocol18Terrain(int protocolVersion, DataTypes dataTypes, IMinecraftCom
         /// <param name="chunksContinuous">Are the chunk continuous</param>
         /// <param name="currentDimension">Current dimension type (0 = overworld)</param>
         /// <param name="cache">Cache for reading chunk data</param>
-        public void ProcessChunkColumnData(int chunkX, int chunkZ, ushort chunkMask, ushort chunkMask2, bool hasSkyLight, bool chunksContinuous, int currentDimension, Queue<byte> cache)
+        public void ProcessChunkColumnData(int chunkX, int chunkZ, long[] chunkMask, ushort chunkMask2, bool hasSkyLight, bool chunksContinuous, int currentDimension, Queue<byte> cache)
         {
+            int chunkMaskSize = chunkMask.Length;
```

Comment:
It seems like this value is not used, so why not passing chunkMask as `long` directly?

Reply:
It seems that this is the preparation of 1.18 behaviour (building height above 256 and below 0). Starting with 1.17 we get an array of long instead of a single long value. So of course it makes sense to handover the array to ProcessChunkColumnData() even if in the moment only the first value is processed.